website/scripts/sync-aragonjs-docs.js

```diff
@@ -63,6 +63,14 @@ const pages = [
 ]
 
 const locationReferenceMap = {
+  'https://github.com/aragon/aragon.js/blob/master/docs/API.md':
+    '/docs/api-js-ref-api.html',
+  'https://github.com/aragon/aragon.js/edit/master/docs/BACKGROUND_SCRIPTS.md':
```

Comment:
Hmm, can we not use just the `/docs/API.md` and `/docs/BACKGROUND_SCRIPTS.md` below for this?